### PR TITLE
Add SQLite support

### PR DESCRIPTION
**Pull request by chalasr**
- [x] Add support for SQLite (~~needs help~~)
-  ~~Update tests~~ (tests still seem to be stuck in the pipes here :trollface:)

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -48,6 +48,7 @@ manala_skeleton_patterns:
       + (manala_skeleton_options.mongodb)|ternary(['mongodb@mongodb' ~ {'3.0': '_3_0', '3.2': '_3_2'}[manala_skeleton_options.mongodb_version|string]],[])
       + (manala_skeleton_options.elasticsearch)|ternary(['elasticsearch@elasticsearch' ~ {'1.5': '_1_5', '1.6': '_1_6', '1.7': '_1_7', '2': '_2'}[manala_skeleton_options.elasticsearch_version|string]],[])
       + (manala_skeleton_options.influxdb)|ternary(['influxdb@influxdata'],[])
+      + (manala_skeleton_options.sqlite)|ternary(['sqlite@manala'],[])
```
Comment (chalasr) [inline vars/main.yml]: really not sure about this one

Comment (nervo) [reply vars/main.yml]: you're absolutely right not to be sure :) don't need this at all as sqlite is simply in the main debian repositories

File: vars/main.yml [hunk 2]
```diff
@@ -144,6 +145,7 @@ manala_skeleton_patterns:
       + (manala_skeleton_options.postgresql)|ternary(['pgsql'],[])
       + (manala_skeleton_options.redis)|ternary(['redis'],[])
       + (manala_skeleton_options.mongodb)|ternary(['mongodb'],[])
+      + (manala_skeleton_options.sqlite)|ternary(['sqlite3'],[])
```
Comment (nervo) [inline vars/main.yml]: sadly, we *have* to handle different php versions, just like
```
      + (manala_skeleton_options.phpmyadmin)|ternary(
        {
          '5.4':[],
          '5.5':[],
          '5.6':[],
          '7.0':['mbstring', 'zip', 'bz2']
        }[manala_skeleton_options.php_version|string],
        []
      )
```

**Comment by chalasr** [comment]
Thanks for your review @nervo, I made the changes. I'll give it some tries tomorrow and keep you informed

**Comment by nervo** [comment]
'sounds good :)
Everything ok for you ? Can i merge ?

If the ansible role itself is ok, we should also tag it. Just tell me.

**Comment by chalasr** [comment]
You can :) We'll quickly see if there's something wrong I guess

**Pull request merged**